BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -13,6 +13,10 @@ private enum ProfileSections: Int {
     case experience = 2
 }
 
+enum screenType {
```

Comment:
- Screen type is a generic name, what happens if you need to config another screen with similar enum?
- Names for enums, classes, structs, subclasses, etc should start capitalize style
- 
rename to `ProfileScreenType`